CODEOWNERS

```diff
@@ -1 +1,9 @@
-/src/lib/gadgets @o1-labs/crypto-eng-reviewers @mitschabaude
+# All files require a review from @o1-labs/o1js-admins
+* @o1-labs/o1js-admins
+
+# Changes to source code should be reviewed by exploration team and future zkapps team 
```

Comment:
I think these rules operate as an 'or', not an 'and'. That is to say, any time someone from one team approves, the other team's review will no longer be required.